code/src/components/TodoList.js

```diff
@@ -0,0 +1,120 @@
+import React from "react";
+import { useSelector, useDispatch } from "react-redux";
+import styled from "styled-components";
+
+import todos from "reducers/todos";
+
+const ListContainer = styled.section`
+  display: flex;
+  flex-direction: column;
+  margin-top: 60px;
+  gap: 10px;
+  align-items: center;
+`
+
+const TodoItem = styled.article `
+  background-color: #aac3bd;
+  padding: 10px;
+  border-radius: 5px;
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  width: 340px;
+
+  h3 {
+    margin: 0;
+  }
+  
+  p {
+    font-size: 12px;
+  }
+
+  input[type='checkbox'] {
```

Comment:
Great to see your code here. I tried to do the customized checkboxes but didn't get it to work. I will try with your code and see if I can solve it. :)